### PR TITLE
[Easy] Use Default Develop Network Settings For Truffle

### PR DESCRIPTION
**Pull request by nlordell**
manually specifying a port is just unnecessary, this PR just removes some configuration and updates code accordingly.

### CODE REVIEW

File: examples/async.rs [hunk 1]
```diff
@@ -12,7 +12,7 @@ fn main() {
 }
 
 async fn run() {
-    let (eloop, http) = Http::new("http://localhost:7545").expect("transport");
+    let (eloop, http) = Http::new("http://localhost:9545").expect("transport");
```
Comment (fleupold) [inline examples/async.rs:15]: why this change?

**Review by fleupold — APPROVED** [review]

File: examples/async.rs (hunk 1 above)
Comment (nlordell) [reply examples/async.rs:15]: So that it works with the `truffle develop` development server which by default is on port 9545. Before `truffle-config` was manually setting a different port but I think its just better to use the defaults instead of maintaining config code.

Comment (nlordell) [reply examples/async.rs:15]: Currently the examples don't depend on ganache(-cli) at all, just truffle.
